.storybook/main.js

```diff
@@ -108,3 +110,25 @@ function overrideDefaultBabelLoader(rules) {
 
   loader.options.customize = customLoaderPath;
 }
+
+/**
+ * @returns {import('storybook-addon-export-to-codesandbox').BabelPluginOptions}
+ */
+function getCodesandboxBabelOptions() {
+  const allPackageInfo = getAllPackageInfo();
+
+  /** @type {import('storybook-addon-export-to-codesandbox').BabelPluginOptions}  */
+  const initialValue = {};
+  return Object.values(allPackageInfo).reduce((acc, cur) => {
+    if (isConvergedPackage(cur.packageJson)) {
+      const prereleaseTags = semver.prerelease(cur.packageJson.version);
```

Comment:
can we simplify this ? maybe to something like:

```js
.reduce((acc, packageInfo) => {
    // not sure if this check is needed, as this config is only for v9
    if (!isConvergedPackage(packageInfo.packageJson)) {
      return acc
   }

   const prereleaseTags = semver.prerelease(packageInfo.packageJson.version) ?? [];
   const packageImport =  prereleaseTags[0].includes('rc') ? '@fluentui/react-components' : '@fluentui/react-components/unstable'

   acc[packageInfo.packageJson.name] = {replace: packageImport }

  return acc;
}, 
{}
)

```

Reply:
it's to avoid having non v9 dependencies in the configuration... it's not that important but I would prefer not to have `@fluentui/react` in the babel plugin config of a v9 storybook addon :)

Reply:
That would actually mean we have to manually change it when we release out of rc. The code currently is more future proof since it applies `react-components/unstable` for anything not rc if it has a prerelease tag